spec/jsSpecs/ares-log.spec.js

```diff
@@ -0,0 +1,385 @@
+            }
+            expect(stdout).toContain("Success", stderr);
+            done();
+        });
+    });
+
+    it("Launch sample App", function(done) {
+        const launchCmd = common.makeCmd("ares-launch");
+        exec(launchCmd + ` ${options.pkgId}`, function (error, stdout, stderr) {
+            if (stderr && stderr.length > 0) {
+                common.detectNodeMessage(stderr);
+            }
+            expect(stdout).toContain(`Launched application ${options.pkgId}`, error);
+            done();
+        });
+    });
+
+    it("Show log with --unit option", function(done) {
+            exec(cmd + " --unit memorymanager", function (error, stdout, stderr) {
+            if (stderr && stderr.length > 0) {
+                common.detectNodeMessage(stderr);
+            } else {
+                const unitExp = /\w+ \d+ \d\d:\d\d:\d\d [\w|\d]+ memorymanager/g;
+                expect(stdout).toContain("-- Logs begin at");
+                expect(stdout.match(unitExp).length > 0).toBeTrue();
+                done();
+            }
+        });
+    });
+});
+
+describe(aresCmd +" -u sam -dp 1", function() {
+    it("Show log with unit and dp option", function(done) {
+            exec(cmd + " -u sam -dp 1", function (error, stdout, stderr) {
+            if (stderr && stderr.length > 0) {
+                common.detectNodeMessage(stderr);
+                expect(stderr).toContain("[Tips]: This device does not support multiple sessions");
+            } else {
+                const unitExp = /\w+ \d+ \d\d:\d\d:\d\d [\w|\d]+ sam/g;
+                expect(stdout).toContain("-- Logs begin at");
+                expect(stdout.match(unitExp).length > 0).toBeTrue();
+            }
+            done();
+        });
+    });
+});
+
+describe(aresCmd + " -S today", function() {
+    it("Show log with --since option", function(done) {
+            exec(cmd + " -S today", function (error, stdout, stderr) {
+            if (stderr && stderr.length > 0) {
+                common.detectNodeMessage(stderr);
+            } else {
+                expect(stdout).toContain("-- Logs begin at");
+            }
+            done();
+        });
+    });
+});
+
+describe(aresCmd + " -U yesterday", function() {
+    it("Show log with --until option", function(done) {
+            exec(cmd + " -U yesterday", function (error, stdout, stderr) {
+            if (stderr && stderr.length > 0) {
+                common.detectNodeMessage(stderr);
+            } else {
+                expect(stdout).toContain("-- Logs begin at");
+            }
+            done();
+        });
+    });
+});
+
+describe(aresCmd + " negative tc", function() {
+    it("Not support option", function(done) {
+            exec(cmd + " -aaa", function (error, stdout, stderr) {
+            if (stderr && stderr.length > 0) {
+                common.detectNodeMessage(stderr);
+                expect(stderr).toContain("[Tips]: Journal daemon do not suppport the option <aaa>");
```

Comment:
Other tc compare expect string starting with "ares-device ERR!". 
`expect(stderr).toContain("ares-device ERR! [Tips]: `

If you think "ares-device ERR" is not needed, please remove it from all expect result.